src/account/default.ts

```diff
@@ -72,18 +60,13 @@ export class Account extends Provider implements AccountInterface {
     const signature = await this.signer.signTransaction(transactions, signerDetails);
 
     const calldata = fromCallsToExecuteCalldataWithNonce(transactions, nonce);
-    const fetchedEstimate = await this.fetchEndpoint(
-      'estimate_fee',
-      { blockIdentifier },
-      {
-        contract_address: this.address,
-        entry_point_selector: getSelectorFromName('__execute__'),
-        calldata,
-        version: toHex(version),
-        signature: bigNumberishArrayToDecimalStringArray(signature),
-      }
+    const fetchedEstimate = await super.getEstimateFee(
+      { contractAddress: this.address, entrypoint: '__execute__', calldata, signature },
+      blockIdentifier,
+      { version }
     );
-    const suggestedMaxFee = estimatedFeeToMaxFee(fetchedEstimate.amount);
+
+    const suggestedMaxFee = estimatedFeeToMaxFee(fetchedEstimate.overall_fee);
```

Comment:
is this overhead already in the overall fee ? or do we need to keep this calculation?

Reply:
Not sure, we can check on the call tomorrow